DSpace/dspace-angular
# Issue#819

## References
* Fixes [https://github.com/DSpace/dspace-angular/issues/819](https://github.com/DSpace/dspace-angular/issues/819)

## Description
This PR fix issue described in [this issue](https://github.com/DSpace/dspace-angular/issues/819) by avoiding that a successful RemoteData is emitted two time, with undefined payload int the first one

## Instructions for Reviewers
To test this PR :
- go to the mydspace page
- open the new submission 
On current main branch scrolling down the list should show an hanging loading icon. With this Pr it should work properly

## Checklist
_This checklist provides a reminder of what we are going to look for when reviewing your PR. You need not complete this checklist prior to creating your PR (draft PRs are always welcome). If you are unsure about an item in the checklist, don't hesitate to ask. We're here to help!_
